docs/configuration/multipledomains.md

```diff
@@ -64,6 +64,92 @@ Multiple top-level domains may also be setup by following the instructions below
          - sub1.alternate2.test:${TRAEFIK_ADDRESS:-0.0.0.0}
          - sub2.alternate2.test:${TRAEFIK_ADDRESS:-0.0.0.0}
     ```
+### Magento 1 Subfolder setup
+If you have a M1 subfolder setup to run your multiple stores, you will have to extend the `nginx` configuration. To do that, add `.warden/warden-env.yml`
+file with the following content to your project root:
+```
+version: "3.5"
+services:
+  nginx:
+    volumes:
+      - ./.warden/nginx/custom.conf:/etc/nginx/default.d/custom.conf
+``` 
+Now you add a file `.warden/nginx/custom.conf` in your project root with the following content:
+```
+location /subdomain1 {
+    alias /var/www/html/;
+    try_files $uri $uri/ @subdomain1;
+
+   location ~ \.php$ {
+       try_files $uri =404;
+       expires off;
+
+       fastcgi_pass $fastcgi_backend;
```

Comment:
I am not really into nginx, but to me, it looks weird that we have to add all these configuration options from the default Magento nginx configuration again. Maybe there is a better way? In a perfect world, we would only add `MAGE_RUN_CODE` here.

Reply:
I tried to remove the other parts, but that always failed.
Maybe someone else has a deeper knowledge of nginx config and could help here?

Reply:
It is possible to simply include an nginx config file to handle the fastcgi stuff. Also very common is to use `echo_exec` to pass the request to php-fpm, which looks very clean.